README.md

```diff
@@ -26,9 +26,40 @@ A large part Ozone Kh's roadmap is managed publicly to promote transparency and
 
 ## Release Notes
 
-### Version 1.7.0 (WIP)
+### Version 1.7.0
+
+- Updated project structure to be compatible with Ozone alpha.10 and above.
+- Removed OCL packages and retire Cancer concept.
+- Updated Common Reports to version 1.4.2
+- Fixed Route translation.
+- Included HPV virus test results in Cancer section in CSS form.
```

Comment:
@kdaud which ticket are you referring for this note?

Reply:
The note is not part of version 1.7.0. I've updated the notes.